ADAL/src/workplacejoin/mac/ADClientCertAuthHandler.m

```diff
@@ -38,36 +51,26 @@ + (void)resetHandler
 {
```

Comment:
In case of a passed-in web view, the window won't necessarily get closed on a timeout (just the web view for example), but in general it's not that important.